src/includes/performance/filter-span-example/javascript.mdx

```diff
@@ -1,8 +1,6 @@
 ```javascript
-import * as Sentry from "@sentry/browser";
-import { Integrations } from "@sentry/tracing";
```

Comment:
This is personal preference, but I'd leave the imports.

Reply:
I felt like, since they're (now it's, as of https://github.com/getsentry/sentry-javascript/pull/3647) in the snippet at the top of the section, it was enough. 

![image](https://user-images.githubusercontent.com/14812505/121029394-6678a280-c75d-11eb-8976-cf89328a3452.png)

I also took out the DSN there, because the point of these lower snippets is really just to highlight the use of the option in question not to be a fully-formed implementation. We do the same thing [here](https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/releases/), for example.